platform/broadcom/sonic-platform-modules-dell/s6000/sonic_platform/sfp.py

```diff
@@ -154,6 +155,15 @@ def _get_eeprom_data(self, eeprom_key):
         if (self.sfpInfo is None):
             return None
 
+        SFP_LOCK_FILE="/etc/sonic/sfp_lock"
```

Comment:
Please consider using python native Lock [https://docs.python.org/3/library/multiprocessing.html#synchronization-between-processes](https://docs.python.org/3/library/multiprocessing.html#synchronization-between-processes). This is more exception friendly and easier to debug deadlock scenarios. Moreover the file lock is originally designed for mutual exclusion for file access.